velox/functions/lib/window/tests/WindowTestBase.cpp

```diff
@@ -48,6 +50,51 @@ WindowTestBase::QueryInfo WindowTestBase::buildWindowQuery(
   return {op, functionSql, querySql};
 }
 
+WindowTestBase::QueryInfo WindowTestBase::buildStreamingWindowQuery(
+    const std::vector<RowVectorPtr>& input,
+    const std::string& function,
+    const std::string& overClause,
+    const std::string& frameClause) {
+  std::string functionSql =
+      fmt::format("{} over ({} {})", function, overClause, frameClause);
+  // split the order by following sql into the orderBy({})
```

Comment:
Please write comments in full sentences... here and in other places in the code-base.

Reply:
Yes.